website/docs/docs/cloud/configure-cloud-cli.md

```diff
@@ -110,13 +110,50 @@ As a tip, most command-line tools have a `--help` flag to show available command
+
+Usage:
+  dbt sqlfluff lint [PATHS]... [flags]
+
+Flags:
+  -d, --dialect string                The dialect of SQL to lint.
+      --encoding string               Specify encoding to use when reading and writing files. Defaults to autodetect.
+  -e, --exclude-rules string          Exclude specific rules. For example specifying -–exclude-rules LT01 will remove rule LT01 (Unnecessary trailing whitespace) from the set of considered rules. This could either be the allowlist, or the general set if there is no specific allowlist. Multiple rules can be specified with commas e.g. –-exclude-rules LT01,LT02 will exclude violations of rule LT01 and rule LT02.
+  -f, --format SqlfluffOutputFormat   What format to return the lint result in, one of "human", "json", "yaml", "github-annotation", "github-annotation-native", "none". (default human)
+  -h, --help                          help for lint
+  -i, --ignore string                 Ignore particular families of errors so that they don't cause a failed run. For example –-ignore parsing would mean that any parsing errors are ignored and don't influence the success or fail of a run. –-ignore behaves somewhat like noqa comments, except it applies globally. Multiple options are possible if comma separated: e.g. –ignore parsing,templating.
+      --ignore-local-config           Ignore config files in default search path locations. This option allows the user to lint with the default config or can be used in conjunction with –config to only reference the custom config file.
+      --rules string                  Narrow the search to only specific rules. For example specifying –-rules LT01 will only search for rule LT01 (Unnecessary trailing whitespace). Multiple rules can be specified with commas e.g. –-rules LT01,LT02 will specify only looking for violations of rule LT01 and rule LT02.
+  -t, --templater SqlfluffTemplater   The templater to use, one of "raw", "jinja", "python", "placeholder", "dbt". (default jinja)
+  -v, --verbose count                 Verbosity, how detailed should the output be. This is stackable, so -vv is more verbose than -v. For the most verbose option try -vvvv or -vvvvv.
+
+Global Flags:
+      --log-format LogFormat   The log format, either json or plain. (default plain)
+      --log-level LogLevel     The log level, one of debug, info, warning, error or fatal. (default info)
+      --no-color               Disables colorization of the output.
+  -q, --quiet                  Suppress all non-error logging to stdout.
+```
 
 #### Considerations
-Keep the following points in mind when using SQLFluff with the dbt Cloud:
+Keep the following points in mind when using SQLFluff with dbt Cloud:
 
 - When you run `dbt sqlfluff`, it picks up changes to your local .sqlfluff config.
 - To use SQLFluff in continuous integration/continuous development, you need to have a `dbt_cloud.yml` file in your project and run commands from a valid dbt project.
```

Comment:
"Sqlfluff commands in the cloud cli currently return an exit code of 0 if the invocation successfully ran - rather than the SQLFluff behavior, where a linting violation returns a non-zero exit code. We plan on addressing this in a later release."

Reply:
Agreed, this is a good call out